static/src/javascripts/projects/common/views/giraffe-message.html

```diff
@@ -1,7 +1,9 @@
 <div class="giraffe">
-    <h2 class="giraffe--heading"><%=linkText%></h2>
-    <p class="giraffe--copy">
-    <%= copy %>
-    </p>
-    <a id="giraffe-contribute" href="<%=linkHref%>" class="button button--primary button--large button--giraffe" data-link-name="<%=linkName%>">Contribute <%= svg %></a>
+    <div class="giraffe--message"/>
```

Comment:
Minor point. We're using `__` for sub-elements: `giraffe__message`, `giraffe__contribute`, etc. and `--` for modifiers: `button--primary`.